frontend/src/app/applications/permit-application-list/permit-application-list.component.html

```diff
@@ -0,0 +1,85 @@
+<section class="usa-section">
+  <div class="usa-grid">
+    <h1>Mount Baker-Snoqualmie National Forest</h1>
+    <div class="application-status-toggle-wrapper">
+      <label for="application-status-toggle">Filter by</label>
+      <select class="usa-input-inline" id="application-status-toggle" (change)="applicationStatusChange($event)">
+        <option value="pending" selected>Pending</option>
+        <option value="accepted">Accepted</option>
+        <option value="rejected">Rejected</option>
+        <option value="cancelled">Cancelled</option>
+        <option value="expired">Expired</option>
+      </select>
+    </div>
+    <h2><span class="permit-application-type">{{applicationStatus}}</span> permit applications</h2>
+    <div *ngIf="apiErrors" class="usa-alert usa-alert-error" role="alert">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">There are errors preventing your data from loading.</h3>
+        <ul class="usa-alert-text">
+          <li *ngFor="let error of apiErrors">{{error}}</li>
+        </ul>
+      </div>
+    </div>
+    <div *ngIf="successMessage" class="usa-alert usa-alert-success application-deadline-warning">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">Success</h3>
+        <p class="usa-alert-text">{{ successMessage }}</p>
+      </div>
+    </div>
+    <div *ngIf="showAttentionAlert()" class="usa-alert usa-alert-warning application-deadline-warning">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">Warning</h3>
+        <p class="usa-alert-text">{{warningMessage}}</p>
+      </div>
+    </div>
+    <div class="application-list">
+
+      <div class="application-field-list-labels">
+        <div>&nbsp;</div>
+        <div *ngIf='isAdmin'>Type</div>
```

Comment:
The isAdmin functionality makes a lot of sense in terms of reusability, but i'm wondering if there is a way to better aggregate those conditionals so it is a bit more readable.

Reply:
We could break these out into two different components, but the result would be markup duplication.